architecture/src/commonTest/kotlin/observable/ReadOnlyPropertyTest.kt

```diff
@@ -31,8 +32,8 @@ class ReadOnlyPropertyTest : ObservableBaseTest() {
     @Test
     fun testReadOnlyPropertyDefaultObservable() = runBlocking {
 
-        var nullableString: String? = null
-        val ro = ReadOnlyProperty<Any?, String?> { _, _ -> nullableString }
+        val nullableString = MutableStateFlow<String?>(null)
```

Comment:
I assume this is because the dispatcher set is no longer respected. Is it still used for anything? in any case the description of the test above is now incorrect.